src/WebAuthn.sol

```diff
@@ -41,40 +38,28 @@ library WebAuthn {
     /// Verifies the authFlags in authenticatorData. Numbers in inline comment
     /// correspond to the same numbered bullets in
     /// https://www.w3.org/TR/webauthn-2/#sctn-verifying-assertion.
-    function checkAuthFlags(
-        bytes1 flags,
-        bool requireUserVerification
-    ) internal pure returns (bool) {
-        // 17. Verify that the UP bit of the flags in authData is set.
+    function checkAuthFlags(bytes1 flags, bool requireUserVerification) internal pure returns (bool) {
+        // 16. Verify that the UP bit of the flags in authData is set.
         if (flags & AUTH_DATA_FLAGS_UP != AUTH_DATA_FLAGS_UP) {
             return false;
         }
 
-        // 18. If user verification was determined to be required, verify that
+        // 17. If user verification was determined to be required, verify that
         // the UV bit of the flags in authData is set. Otherwise, ignore the
         // value of the UV flag.
-        if (
-            requireUserVerification &&
-            (flags & AUTH_DATA_FLAGS_UV) != AUTH_DATA_FLAGS_UV
-        ) {
+        if (requireUserVerification && (flags & AUTH_DATA_FLAGS_UV) != AUTH_DATA_FLAGS_UV) {
             return false;
         }
 
-        // 19. If the BE bit of the flags in authData is not set, verify that
```

Comment:
Not sure what this comment refers to, as I believe these bits are reserved for future use https://www.w3.org/TR/webauthn-2/#authenticator-data